### PR TITLE
Added contribution guide

### PR DESCRIPTION
**Pull request by KareemMAX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 Unify database is a collection of curated YAML files that hold a description of different algorithms, tools, repos, products, related to different layers of the ML deployment stack, mainly: orchestration, compression, compilers, hardware.
 
+**[Website](https://unify.ai/database)** - **[Contribute](CONTRIBUTING.md)**
+
+## How to access our database
```
Comment (guillesanbri) [inline README.md:7]: maybe better to use **the** database imo

File: README.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+
+You can offer the database through:
+
+- [Our website](https://unify.ai/database).
```
Comment (guillesanbri) [inline README.md:11]: same here, "our" sounds a bit too corporative imo, maybe better to use "the website" or "its website"

File: SECURITY.md
Comment (guillesanbri) [inline SECURITY.md:1]: do you reckon we need this file? doesn't seem to add too much information, not strongly opinionated against it tho

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Removing an entry
+
+1. Create an issue detailing the reason for the removal
+2. We will remove the entry and close the issue
+
+## Report an issue
+
+1. Create an issue detailing the issue
+2. Please include the link to the entry, as well as the reason for the issue
+3. We will fix the issue and close the issue
```
Comment (guillesanbri) [inline CONTRIBUTING.md:33]: I'd either remove this line or change it to something like "We will look into it!"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+## Updating an existing entry
+
+1. Fork this repository
+2. Update the `.yaml` file
+3. Create a pull request detailing the changes
+
+## Removing an entry
+
+1. Create an issue detailing the reason for the removal
+2. We will remove the entry and close the issue
```
Comment (guillesanbri) [inline CONTRIBUTING.md:27]: maybe worth adding something like "we will evaluate the request and remove the entry if needed"

**Review by guillesanbri — CHANGES_REQUESTED** [review]
Minor comments but lgtm, we will have to work on a pipeline for suggestions on entirely new cards with an issue or a google form, but no need to do this before merging this pr imo

File: SECURITY.md
Comment (KareemMAX) [reply SECURITY.md:1]: Not really sure, I'm adding it as it was suggested by [Github](https://github.com/unifyai/database/community). And we might suffer from attacks as well (although I'm pretty confident that would be really rare). I'm not strongly opinionated to keep it either, but I think it won't really hurt.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -24,10 +24,10 @@ In both cases, we will upload the image to our servers and update the link in th
 ## Removing an entry
 
 1. Create an issue detailing the reason for the removal
-2. We will remove the entry and close the issue
+2. We will will evaluate the request and remove the entry if needed
```
Comment (guillesanbri) [inline CONTRIBUTING.md:27]: duplicated ``will`` 😅

File: SECURITY.md
Comment (guillesanbri) [reply SECURITY.md:1]: sounds good let's leave it then

File: README.md [hunk 6]
```diff
@@ -2,13 +2,13 @@
 
 Unify database is a collection of curated YAML files that hold a description of different algorithms, tools, repos, products, related to different layers of the ML deployment stack, mainly: orchestration, compression, compilers, hardware.
 
-**[Website](https://unify.ai/database)** - **[Contribute](CONTRIBUTING.md)**
+**[The Database](https://unify.ai/database)** - **[Contribute](CONTRIBUTING.md)**
 
 ## How to access our database
```
Comment (guillesanbri) [inline README.md:7]: I meant in the header, the previous ``website`` in the link was good :D

**Review by guillesanbri — COMMENTED** [review]
couple of things, feel free to merge after these, ty!

**Pull request merged**